docs/structure-of-a-contract.rst

```diff
@@ -11,6 +11,11 @@ Each contract can contain declarations of :ref:`structure-state-variables`, :ref
 :ref:`structure-function-modifiers`, :ref:`structure-events`, :ref:`structure-struct-types` and :ref:`structure-enum-types`.
 Furthermore, contracts can inherit from other contracts.
 
+There are also special kinds of contracts called :ref:`libraries<libraries>` and :ref:`interfaces<interfaces>`.
+
+The section about :ref:`contracts<contracts>` will go much more into detail than this section,
```

Comment:
``will go into much more detail than this section``? Or even ``contains much more details than this section``?